stable/insights-agent/templates/kyverno/rbac.yaml

```diff
@@ -14,20 +14,29 @@ metadata:
     app: insights-agent
 rules:
 - apiGroups:
-  - '*'
+  - kyverno.io
```

Comment:
I'm indifferent if we want to wildcard the whole group or not, but we will also need to include the `wgpolicyk8s.io` api group

```suggestion
  - kyverno.io
  - wgpolicyk8s.io
```

Reply:
That's for these CRDs:

```
clusterpolicyreports.wgpolicyk8s.io
policyreports.wgpolicyk8s.io
```